project.janet

```diff
@@ -9,22 +9,26 @@
   :name "rlrepl"
   :source ["rlrepl.janet"])
 
-(defn pkg-config [what]
-  (def f (file/popen (string "pkg-config " what)))
-  (def v (->>
-           (file/read f :all)
-           (string/trim)
-           (string/split " ")))
-  (unless (zero? (file/close f))
+(defn exec_get_string_and_exit_code [args]
+  (def env (os/environ))
+  (def streams (os/pipe))
+  (put env :out (streams 1))
+  (def exit_code (os/execute args :pe env))
```

Comment:
The current code only works if the output of the command is less than PIPE_BUF.  iirc the correct way is to use spawn, read then wait for the exit code.

Reply:
I just pushed a commit that addresses that.  
Thanks for the insight

Reply:
I think your new commit uses exit_code but doesn't define it.